packages/ui/src/components/Command/DocsSearch.tsx

```diff
@@ -205,7 +210,7 @@ const DocsSearch = () => {
                         openLink(page.type, formatSectionUrl(page, section))
                       }}
                       key={`${page.meta.title}__${section.heading}-item-index-${i}`}
-                      value={`${page.meta.title}__${section.heading}-item-index-${i}`}
+                      value={`${removeDoubleQuotes(page.meta.title)}__${section.heading ? removeDoubleQuotes(section.heading) : ""}-item-index-${i}`}
```

Comment:
```suggestion
                      value={`${removeDoubleQuotes(page.meta.title)}__${removeDoubleQuotes(section.heading ?? '')}-item-index-${i}`}
```

Reply:
you could shorthand it like this maybe? :)

Reply:
Will update it

Reply:
If you update that part then we can merge. Thanks, @YugBhanushali 👍

Reply:
Nice, thanks!